Dalamud/Interface/ImGuiFileDialog/ImGuiFileDialogFlags.cs

```diff
@@ -16,41 +16,41 @@ public enum ImGuiFileDialogFlags
         /// <summary>
         /// Confirm the selection when choosing a file which already exists.
         /// </summary>
-        ConfirmOverwrite = 1,
```

Comment:
Are these breaking changes?

Reply:
Technically, yes. But a) as far as I know nobody uses those flags at all yet (the manager handles them for you anyway), and b) if they are, they're absolutely not working as expected, in that you can't actually use them as flags and any try to do so will result in very weird consequences.
But yeah, breaking - but they have to be.